components/webgpu/lib.rs

```diff
@@ -371,7 +376,14 @@ impl<'a> WGPU<'a> {
             receiver,
             sender,
             script_sender,
-            global: wgpu::hub::Global::new("wgpu-core", factory, wgt::BackendBit::PRIMARY),
+            global: wgpu::hub::Global::new(
+                "wgpu-core",
+                factory,
+                InstanceDescriptor {
+                    backends: wgt::Backends::PRIMARY,
+                    dx12_shader_compiler: Dx12Compiler::default(),
```

Comment:
Will this work on non-Windows backends?

Reply:
Yes, this option is ignored on non-windows platforms.